README.md

```diff
@@ -17,6 +17,7 @@ discount - The discount percentage to be applied to the converstion rate
 
 You can see a running example implementation here: 
 
+(EXAMPLE NO LONGER UP TO DATE - LEAVING HERE FOR REFERENCE)
```

Comment:
It's up to the repo author to do what he wants with proposed changes.  i just placed that on my branch because the demo does not have these changes implemented, so its not functioning - just a note to self